### DISCUSSION TITLE
Should we restrict the `gap` prop values in the Flex component?

### DISCUSSION THREAD

**Discussion by GomezIvann** [dxc-technology/halstack-react#1427] [Ideas] [answered]
After using the `DxcFlex` component for a while and in various situations, I found myself always using the [spacing values from our design guidelines](https://developer.dxc.com/halstack/next/principles/spacing/#core-spacing-tokens) and trying to follow them as much as possible. So, I question whether we should restrict or limit the values from the `gap` prop. Do we want the users to use other values different from the ones we encourage them to use? In which use cases that would be a fair proposal/approach? Are they valid?

**Comment by kukoc** [comment, marked as answer]
It makes a lot of sense to me to align the allowed values of the gap prop with the spacing values in our guidelines. I don't know, looks like very reasonable. Users get the IntelliSense of typescript so the don't choose a wild value that creates inconsistency in our uis.

**Reply by GomezIvann** [reply]
I'm moving this to an issue since we are currently working on the Layout components and the next release could be a good moment to address this!

**Comment by GomezIvann** [comment]
Moving this discussion to an issue #1568 🚀
